.gitignore

```diff
@@ -0,0 +1,11 @@
+*.aux
+*.fdb_latexmk
+*.fls
+*.log
+*.out
+*.synctex.gz
+*.bbl
+*.bcf
+*.blg
+*.run.xml
+*.toc
```

Comment:
Thanks for adding a `.gitignore`! Surely there was a reason I didn't add one 6 years ago but I can't think of one now :)

Reply:
I didn't actually mean to add that. I'm happy to add it in another PR. WDYT?